### PR TITLE
Feature/pretty

### LINKED ISSUE #276: Loading Indicator on Login Page
https://android-arsenal.com/details/1/6394

The login screen loads the REST database to make sure the credentials are correct; show a loading indicator while the credentials are being tried

### PR DESCRIPTION
**Pull request by bhylak**
Made a bunch of stylistic changes, better animations, white/grey theme on survey page, etc.

Fixes #276

### CODE REVIEW

**Comment by krconv** [comment]
Is this tested on API 19?

**Comment by bhylak** [comment]
Yes.

File: app/src/main/java/org/fundacionparaguaya/advisorapp/activities/DashActivity.java [hunk 1]
```diff
@@ -116,22 +114,30 @@ protected void onCreate(Bundle savedInstanceState) {
...
         mSyncArea = findViewById(R.id.linearLayout_topbar_syncbutton);
         mSyncArea.setOnClickListener(this::onSyncButtonPress);
-
         mSyncButtonIcon = findViewById(R.id.iv_topbar_syncimage);
-
         mBackButton = findViewById(R.id.linearlayout_dashactivity_back);
         mBackButton.setVisibility(View.GONE);
         mBackButton.setOnClickListener((event)-> onBackPressed());
 
+       // mTvTabTitle.setFactory(mFactory);
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/activities/DashActivity.java]: Think this should be removed

File: app/src/main/java/org/fundacionparaguaya/advisorapp/activities/DashActivity.java [hunk 2]
```diff
@@ -116,22 +114,30 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        Animation inAnim = AnimationUtils.loadAnimation(this,
+                android.R.anim.fade_in);
+        Animation outAnim = AnimationUtils.loadAnimation(this,
+                android.R.anim.fade_out);
+
+        inAnim.setDuration(100);
+        outAnim.setDuration(100);
+
+       // mTvTabTitle.setInAnimation(inAnim);
+       // mTvTabTitle.setOutAnimation(outAnim);
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/activities/DashActivity.java]: Is this whole animation creation ever used?

File: app/src/main/java/org/fundacionparaguaya/advisorapp/activities/DashActivity.java (hunk 1 above)
Comment (bhylak) [reply app/src/main/java/org/fundacionparaguaya/advisorapp/activities/DashActivity.java]: resolved.

**Comment by bhylak** [comment]
resolved comments @krconv

**Comment by krconv** [comment]
image
The color on the priorities page should be fixed

Also, when a new survey is taken (both a resurvey and a new family survey), screen flashes yellow (at least on api 19)
image

**Comment by krconv** [comment]
image
When the login is incorrect, the login fields aren't reenabled

**Pull request merged**